tests/api/BUILD

```diff
@@ -16,6 +16,7 @@ ts_test_suite(
         "//examples/invalid_dataform_json:node_modules",
         "//examples/never_finishes_compiling:files",
         "//examples/never_finishes_compiling:node_modules",
+        "//sandbox/vm:worker_bundle",
```

Comment:
This shouldn't be here, it should come through the dep on `//api`

Reply:
I've have a play with this, and some alternative structures. I think you're right, it should come through from `//api`. However we can't include it as a `dep` of `//api` because the worker bundle is plain javascript and has no types, so we have to put it as `data`. From this, I think that the real issue here is therefore that `ts_test_suite` does not include transitive `data` dependencies from it's typescript `dep` rules, and so we should just include the explicit `worker_bundle` `data` rule in `//tests/api` and `//tests/integration`. WDYT?

Reply:
The tests actually pass without the worker bundle being in `//api`, but I think we need that there for the CS

Reply:
Sounds good, yup this is fine! Getting it out of the CLI test was the important bit :)